docs/scarpet/api/Entities.md

```diff
@@ -693,6 +717,34 @@ Applies status effect to the living entity. Takes several optional parameters, w
 `true` and `false`. If no duration is specified, or if it's null or 0, the effect is removed. If name is not specified,
 it clears all effects.
 
+### `modify(e, 'may_fly', boolean)`
+
+Sets the value of the player's `mayfly` (The ability of the player to fly) ability to the given boolean value.
+
+### `modify(e, 'flying', boolean)`
+
+Sets the value of the player's `flying` (If the player is flying or not) ability to the given boolean value.
+
+### `modify(e, 'may_build', boolean)`
+
+Sets the value of the player's `mayBuild` (The ability of the player to place blocks) ability to the given boolean value.
+
+### `modify(e, 'insta_build', boolean)`
+
+Sets the value of the player's `instabuild` (The ability of the player to place blocks without reducing their count in the inventory and shoot with a bow without arrows) ability to the given boolean value.
+
+### `modify(e, 'fly_speed', float)`
+
+Sets the value of the player's `flySpeed` (The speed at which the player moves while flying) ability to the given float value.
+
+### `modify(e, 'walk_speed', float)`
+
+Sets the value of the player's `walkSpeed` (The speed at which the player moves while walking) ability to the given float value.
```

Comment:
This is just aesthetic and very personal opinion, but I think it would be nice to have those (and the queries) inside an `ability` or `abilities` "section" (so it would end up `modify(e, 'ability', 'walk_speed')`).

To do those I _think_ you just have to expect a `ListValue` inside the lambda of the `ability`/`abilities` map key instead of a single value, should just add a switch on string in there.

What do you think?

Reply:
I'll fix all these thing tomorrow. Thank you for the review!

Reply:
abilities is an internal thing. From the outsider, its yet another attributes of players in particular. Hence I don't think there is a need of grouping them under 'abilities'

Reply:
So, is it ok as it is now, with a modify action for each of the abilities?